go/parquet/pqarrow/file_writer.go

```diff
@@ -181,7 +188,10 @@ func (fw *FileWriter) WriteBuffered(rec arrow.Record) error {
 }
 
 // Write an arrow Record Batch to the file, respecting the MaxRowGroupLength in the writer
-// properties to determine whether or not a new row group is created while writing.
+// properties to determine whether a new row group is created or not while writing.
+//
+// Performance-wise Write might be more favorable than WriteBuffered
+// especially if dealing with records that have a lot of rows.
```

Comment:
Let's mention that this would be more performant in the case of a highly restricted memory environment, or if you have very large records with lots of rows, potentially close to the `max row group length`.

Reply:
Done in 9c4ca94137a143986e69feb07a0ab94baee7e334